en/identity-server/next/docs/includes/db-config-table.md

```diff
@@ -38,3 +38,33 @@ The elements in the above configuration are described below:
 !!! info 
     For more information on other parameters that can be defined in
     the `<IS_HOME>/repository/conf/deployment.toml` file, see [Tomcat JDBC Connection Pool](http://tomcat.apache.org/tomcat-9.0-doc/jdbc-pool.html#Tomcat_JDBC_Enhanced_Attributes).
+
+??? note "Support for Case Sensitive Usernames"
+
+    If you are going to have case-sensitive usernames, 
+    you need to add the following properties to the deployment.toml file.
```

Comment:
Remove

Reply:
resolved with https://github.com/wso2/docs-is/pull/4209/commits/24ec8d148e9a0a8fa9efc843b02d9856a987e914